dockerBasedBuild/dockerBasedBuild.common.psm1

```diff
@@ -579,7 +578,7 @@ class BuildData
     [String]$BinaryBucket = 'release'
 
     # Optional: the number of expected binaries from the build.
-    [Int]$BinariesExpected = -1
+    [Int]$ArtifactsExpected = -1
```

Comment:
This requires JSON changes.  Make sure to update the VSCode repo

Reply:
The `build.json` file in `vscode-powershell` repo doesn't contain the `BinariesExpected` key.